app/assets/javascripts/tables/base/editable_table.js

```diff
@@ -34,9 +34,17 @@ var EditableTable = (function () {
     function extractTextfromCells(row) {
         return $(row).find("td.editable").toArray().map(function (cell) {
             var input = $(cell).find("select:visible").length > 0 ? 'select' : 'input',
-                elem  = $(cell).find(input);
+                elem  = $(cell).find(input),
+                type  = elem.data('type'),
+                value = $.trim(elem.rawValue());
+
+            if (type === 'float') {
+                value = parseFloat(value);
+            } else if (type == 'integer') {
+                value = parseInt(value);
+            }
```

Comment:
[The topology form uses `type=number`](https://github.com/quintel/etmoses/blob/2c874f951c0903e6d305859c85ca547d4ec05f0b/app/assets/javascripts/topology/form.js#L20-L22), but here we're using `float/integer`. Is it perhaps worth making these names consistent?

I think it would be safe to use `type === 'number' / parseFloat` even for those fields which expect an integer, since `InstalledTechnology` will convert those values to integers anyway. Alternatively if you do want to distinguish between fields which may have a fraction and those which may not, then maybe the topology form should also use `type=float`?